src/app/clusters/ota-requestor/ota-image-processor.h

```diff
@@ -23,12 +23,12 @@
 
 #pragma once
 
```

Comment:
Wouldn't it be also beneficial to add all includes so that `CHIP_ERROR`, `ByteSpan`, and `chip::Optional` are resolved?